package.json

```diff
@@ -39,7 +39,8 @@
     "test"
   ],
   "bin": {
-    "hermione": "./bin/hermione"
+    "hermione": "./bin/hermione",
+    "testplane": "./bin/hermione"
```

Comment:
You still have one binary file, so `npx testplane` will launch `./bin/hermione`, and `{cliName: 'hermione'}` would be passed to `run` method either way.

Reply:
Fixed: `"testplane": "./bin/tesplane"`.